service/datastore/internal/src/main/resources/kapua-datastore-settings.properties

```diff
@@ -32,3 +32,7 @@ datastore.cache.metadata.local.size.maximum=1000
 
 # Datastore index prefix
 datastore.index.prefix=
+
+#
+#maximum value for limit parameter
+datastore.limit.max_value=10000
```

Comment:
I think that `datastore.query.limit.value.max` o `datastore.query.limit.max` would be better for the scope of the setting